pyproject.toml

```diff
@@ -17,31 +20,31 @@ python = ">=3.9,<3.14"
 click = "*"
 platformdirs = "*"
 PySide6 = "^6.7.1"
-PyMuPDF = "^1.23.3"  # The version in Fedora 39
+PyMuPDF = "^1.23.3"                           # The version in Fedora 39
```

Comment:
Out of curiosity, what's with the extra spaces? Did a linter change this?

Reply:
:astonished: I don't remember doing this, and I don't use AI on this codebase, so... I don't know! :roll_eyes:

I'll fix this, thanks.